corehq/apps/data_dictionary/views.py

```diff
@@ -97,17 +116,36 @@ def data_dictionary_json(request, domain, case_type_name=None):
 @toggles.DATA_DICTIONARY.required_decorator()
 def update_case_property(request, domain):
     property_list = json.loads(request.POST.get('properties'))
+    fhir_resource_type = request.POST.get('fhir_resource_type')
+    fhir_resource_type_obj = None
+    case_type = request.POST.get('case_type')
     errors = []
-    for property in property_list:
-        case_type = property.get('caseType')
-        name = property.get('name')
-        description = property.get('description')
-        data_type = property.get('data_type')
-        group = property.get('group')
-        deprecated = property.get('deprecated')
-        error = save_case_property(name, case_type, domain, data_type, description, group, deprecated)
-        if error:
-            errors.append(error)
+
+    if request.POST.get('remove_fhir_resource_type', '') == 'true':
```

Comment:
note to self: the two ifs along with the part of checking for props on request object can come in a block of if toggles.FHIR_INTEGRATION.enabled check or infact be extracted out of this method entirely